lib/inspec/resources/parse_config.rb

```diff
@@ -55,8 +55,11 @@ def initialize(content = nil, useropts = nil)
       read_params unless @content.nil?
     end
 
-    def method_missing(name)
-      read_params[name.to_s]
+    def method_missing(*name)
+      # catch bahavior of rspec its implementation
+      # @see https://github.com/rspec/rspec-its/blob/master/lib/rspec/its.rb#L110
```

Comment:
This URL is already bad - consider to replacing with a permalink

Reply:
Fixed by making it point to tag v1.2.0, the version of rspec-its currently in use.